### ISSUE TITLE
[CAIA A11y Research]: Authenticated Experience Design Patterns Team, Update Prefill Pattern (Nov. 2024)

### ISSUE THREAD

**Issue by ChristineRoseSteiffer** [department-of-veterans-affairs/va.gov-team#94733]
> [!NOTE]
> **High-Level Overview**: Update Prefill Pattern
> - **Initial Meeting Requested:** [Wednesday 10/16 at 12 p.m. EST](https://dsva.slack.com/archives/C8E985R32/p1728568230052909?thread_ts=1728507763.664619&cid=C8E985R32) `Scheduled`
> - **Research Timeframe**:  _AT Sessions:_ 11/15-11/20/2024
> - **Pilot**: #95123 
> - **Assistive Tech Users**: `5` AT Users
> - **Artifacts Available**: [Research Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/authenticated-patterns/Design%20and%20Research/2024-09%20Research%20Initiative%202%20-%20Update%20Prefill/Update%20Prefill%20Research%20Plan.md) and [Convo Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/authenticated-patterns/Design%20and%20Research/2024-09%20Research%20Initiative%202%20-%20Update%20Prefill/Update%20Prefill%20Research%20Conversation%20Guide.md)
> - **Testing Environment & Schedule:** Codespace Test Environment [prototype link](https://effective-space-tribble-rq7q67g6gv359p5-3001.app.github.dev/mock-form-ae-design-patterns/2) and Source of Truth [Prefill Research Study - Observer Spreadsheet.xlsx](https://dvagov.sharepoint.com/:x:/s/TMFAuthenticatedExperienceDesignPatterns/ESNQw6tGb8tDmv3iiv7f-TgBLnfMvhPFKjy6HNsMe-mmdw?e=Mch2EH) - this is sharepoint so you need to be on the VPN
> - **Assistive Tech Being Used**: `Magnifier` `Fusion` `JAWS` `TalkBack` `VoiceOver`
> - **CAIA Internal Doc:** [**CAIA Raw Session Notes: Google Doc**](https://docs.google.com/document/d/1lQzJB0q0ePFumxC749K-58lVhaAZh7_OZPhR-uU9Cuk/edit?tab=t.0)
> - **Task Lists**: See task lists for next steps/definition of done

## CAIA Assignee(s)

<details><summary>Toggle to view information about the CAIA Team.</summary>
- PIlot: #95123 
    -  @mutiaadhocteam would like to assist with the pilot. 
    - **Pilot (Desktop):** `JAWS` CodeSpace
- 
- **Research Session Support:**
    - @sara-amanda `October 18 – 25, 2024`
    - @mutiaadhocteam `OOO October 18 – 25, 2024`
    - @coforma-jamie 
    
</details>

## About the Team

- Team name: Authenticated Experience Design Patterns (AEDP)
- OCTO-DE product owner: @beckyphung 
- Product manager: @msbtterswrth 
- Designer: @ChristineRoseSteiffer + @kristen101606 
- FE engineer: @adamwhitlock1  + @bellepx0 
- Product/team Slack channel: #tmf-auth-exp-design-patterns

<details><summary>Toggle to view more intake research ticket information</summary>

### Do you have an accessibility champion?

No, we don't have a champion (yet)

### Calendar checking

- [x] Based on this calendar, confirm we have specialists who can support your study in your desired timeframe
- [x] Confirm you are filing this support ticket at least 1 week in advance of said timeframe, so we have enough heads up to provide support

### Have you reviewed the CAIA capacity calendar?

- [x] Yes, we have reviewed the CAIA capacity calendar and can confirm specialists are available to support in our research timeframe

</details>

### Initial Request

> Hey CAIA team! The AEDP is back with our next pattern, Update Prefill. We are still working through cleaning up our designs and writing our research documentation this sprint, but we wanted to get on y'alls calendars and radars ASAP to help us work through a few key issues we're identified with our pattern. Those include:
> 
> - Prefill Alert text - specifically, ensuring people have a clear understanding of where their changes will be saved if they update their prefilled data
> - Text advising users how to update locked data such as name, DOB, etc and if that should be behind and 'additional info' component
> - Phrasing of a new question we are testing asking users where they want to save changes (to just the form or to the profile as well?)
> 
> Would we be able to meet either Wednesday or Friday next week to have an initial discussion about these topics? Thank you!

#### Intake Ticket Reference to this Research

> Our next pattern is about Updating Prefilled Information, so we were planning on trying to recruit AT users for that study. We will test our recommendations for Prefill as well as some variations related to updating that info. ...

### Previous Study Findings 

<details><summary>Toggle to view details.</summary>

#### AEDP Prefill Study 9/30/2024
[Reference the previous study's research findings](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/authenticated-patterns/Design%20and%20Research/2024-07-Research%20Initiative-One-Prefill/Prefill%20Research%20Report%2009_2024.md)

> **Our team has been tasked with:**
> - creating 4 - 8 reusable patterns for personalized, authenticated experiences,
> - making it as easy as possible to implement across products and teams and
> - creating documentation for personalization patterns ready for integration into both the VA Design System (VADS) and the US Web Design System (USWDS).

#### Key Prefill findings
**AEDP Prefill Study | 09/30/2024**

1. Veterans donʼt understand what is meant by ʻprofile dataʼ and some conflate VA.gov data with their method of authentication (Login.gov, ID.me or MHV).
2. Veterans want to see the prefill alert at the top of the Intro page, on the prefill confirm page, and on the edit prefilled data page.
3. Bolding the word ʻNoteʼ at the beginning of an important text block draws more attention to the information.
4. There was a slight preference of the gray card style over the address component style, but the results are not conclusive.
5. Understanding why they canʼt update certain information on the site helps Veterans to build trust with the VA.

#### Secondary findings related to Update Prefill
**AEDP Prefill Study | 09/30/2024**

1. While most people expected their updates to save to their profile, many could think of examples of when they might want to choose whether the update saves to the just form or to their profile as well.
2. While some people wanted the ability to edit on the same page, most people expected the ʻEditʼ button to take them to a new page.
3. The option to edit ʻoldʼ data within the form field itself was widely preferred on the edit page.

#### Additional findings related to Update Prefill
**AEDP Prefill Study | 09/30/2024**

1. It is not always clear to people the difference between the autosave feature and updates to their prefilled data.
3. The process of signing in is difficult and confusing.
4. Sharing a prototype on mobile is hard.

</details>

## Research Sessions

> [!TIP]
> - Providing CAIA A11ys with the time slots in advance for AT users, will allow a11ys to avoid conflicts on the CAIA A11y Capacity Calendar, and reach out to other a11ys, should CAIA not be available. 
> - Providing CAIA A11ys with the technology and devices the users will be testing with, will allow CAIA to prepare in advance for the sessions. 

### Timeframe

<details><summary>Toggle to view research timing information.</summary>

#95123 
1. **Pilot Session: 11/6/2024 at 1:30pm EST - JAWS

End of October / Early November 2024
- **Pilot:** Week of 10/28/2024 (per 10/16/2024 meeting)
- **Testing**: ~11/12-11/15/2024 (per 10/16/2024 meeting)~ 11/15 - 11/20/2024 per Slack 11/6/2024.

### Length of sessions

- **Session length:**
    - `90 Min.` 
    - **AT Session Timing Examples**: 
        - `90` minutes (screen reader or alternative navigation tool)
        - `60` minutes (screen magnification tool)
        - **Buffer time between sessions:** `30` minutes
        - **Maximum sessions per day:** `2`

</details>

###  Testing Schedule

CAIA A11Ys will be looking for device and AT information. We realize that some of this information will not be available until the day of the session. 

<details><summary>Toggle to view time-related items</summary>

#### Enter Testing Information
- **Dates**: Nov. 15, 2024 to `Nov. 20, 2024`
- **Times**:  `See Table`
- **Planning to Recruit `#` Veterans for**:
     - minimum of `#` sessions with screen reader users
     - minimum of `#` sessions with alternative navigation users
     - minimum of `#` session with a screen magnification user

#### Examples
- **Date Range Example**: _Typically around `2` Weeks_ **11/12-11/15/2024 (per 10/16/2024 meeting)**
- **Timing Example**: _`9-10:30am`, `11-12:30pm`, `1-2:30pm`, `3-4:30pm` EST_
- **Recruitment Example**: _Planning to Recruit `12` Veterans for: a minimum of `3` sessions with screen reader users; minimum of `2` sessions with alternative navigation users; and a minimum of `1` session with a screen magnification user._

#### Source of Truth
Testing Schedule [Update Prefill Research Study - Observer Spreadsheet.xlsx](https://dvagov.sharepoint.com/:x:/s/TMFAuthenticatedExperienceDesignPatterns/ESNQw6tGb8tDmv3iiv7f-TgBLnfMvhPFKjy6HNsMe-mmdw?e=Mch2EH) - this is sharepoint so you need to be on the VPN;  **_also see table in this ticket_**

Please use the following table to enter the following information for each participant: Date and Time, Assistive tech (AT), Device and Operating System, Browser, and Skill level:

P# | Date and Time (ET) | AT | Device and OS | Browser | Skill | CAIA A11ys Available
------------------|------------------|--------------|--------------|--------------|--------------|--------------
Pilot `#95123` | **Week of 10/28/24**, 1:30 p.m. |JAWS | Desktop | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | @sara-amanda @coforma-jamie @mutiaadhocteam 
P8 | ~~11/18/24, 11-12:30pm~~ `no show` | Fusion | Mobile or Desktop; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | @coforma-jamie 
P9 | 11/18/24, 2-3:30pm | Pinch and zoom | Mobile; iOS | Firefox | Novice, Intermediate, Advanced `TBD` | @coforma-jamie  
P10 | 11/19/24, 11-12:30pm | Fusion (ZoomText/JAWS)| Desktop; Windows | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | @SarahKay8 
P11 | 11/20/24, 11-12:30pm | TalkBack | Mobile  Android | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | @mutiaadhocteam 
P12 | 11/20/24, 1-2:30pm | VoiceOver | Mobile or Desktop;  MacOS, iOS | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | @mutiaadhocteam 

- **Consideration**: You can also consider asking participants more specific AT and device combinations. 
- **For instance:** TalkBack on Samsung Galaxy 8, or VoiceOver on iPad with magnification.

</details>

CAIA A11ys will review the coverage needed, and self-assign, based on a11y availability.

### Supporting artifacts (if any)

- [Product outline](https://docs.google.com/presentation/d/1wuRokQJ6EPotKCMKcI3OX41U8rGgJC2Ehvn7W3RMXi8/edit#slide=id.g268fb482a95_0_65)
- [Draft Designs - WIP](https://www.figma.com/design/1z3bAkQl4uR1IvAxmtyqZi/AE-Design-Patterns---Update-Prefill?node-id=3089-14103&node-type=frame&t=eKEIsDwB3ICWzIbT-0)
- [Research Plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/authenticated-patterns/Design%20and%20Research/2024-09%20Research%20Initiative%202%20-%20Update%20Prefill/Update%20Prefill%20Research%20Plan.md)
- [Convo Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/authenticated-patterns/Design%20and%20Research/2024-09%20Research%20Initiative%202%20-%20Update%20Prefill/Update%20Prefill%20Research%20Conversation%20Guide.md)
- [Codespace prototype link](https://effective-space-tribble-rq7q67g6gv359p5-3001.app.github.dev/mock-form-ae-design-patterns/2)

## Next Steps: Definition of Done

```[tasklist]
### CAIA A11Y Tasks
- [x] **INTAKE TICKET:** @sara-amanda to process intake ticket 10/10/2024
- [x] **OUTREACH**: Initial Meeting Requested by Auth Exp. Design Patterns Team for 10/16 or 10/18/2024.
- [x] **INITIAL MEETING:** Schedule an initial meeting with team to discuss the research process and support. `Scheduled`
- [x] **Links Needed and Received**: Research Guide and Convo Guide Support
- [x] **PILOT CALENDAR HOLDS** - on the A11y [**CAIA Capacity Calendar**](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/Chicago) `Based on Initial Meeting` #95123
- [x] **PILOT SESSION DESKTOP: SCHEDULE** Pilot session with team.
- [x] **PILOT SESSION DESKTOP: CREATE TICKET** See `related tickets task list` in this ticket.
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/95123
- [x] **CALENDAR HOLDS** Update A11y [CAIA Capacity Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/Chicago) with holds for user research sessions with AT, when dates become available. Updated on 10/16/2024 with new research dates.
- [x] **Receive Source of Truth for Scheduling** 🔗 (Examples: Google Sheet, Slack Canvas, etc.)[Update Prefill Research Study - Observer Spreadsheet.xlsx](https://dvagov.sharepoint.com/:x:/s/TMFAuthenticatedExperienceDesignPatterns/ESNQw6tGb8tDmv3iiv7f-TgBLnfMvhPFKjy6HNsMe-mmdw?e=Mch2EH) - this is sharepoint so you need to be on the VPN
- [x] **Divvy user research session support with CAIA A11ys** - based on availability and technology available to support. `Completed 11/6/2024`
- [x] **Updated Holds:** waiting on official invites (table to be updated in this ticket) `Completed 11/6/2024`
- [x] **Receive prototype from Team for testing in:**  [Codespace prototype link](https://effective-space-tribble-rq7q67g6gv359p5-3001.app.github.dev/mock-form-ae-design-patterns/2) `RCVD 11/6/2024`
- [x] **Research Tracker (CAIA Internal)** `Updated 11/6/2024` - reflects session holds and AT
- [x] **REMOVE CALENDAR HOLDS** Remove holds on A11y [**CAIA Capacity Calendar**](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/Chicago) and replace with invitations from project team (or Perigean) `Updated holds 11/6/2024` `Removed holds 11/8/2024`
- [ ] **Requested device breakdown:** Desktop, Mobile and Tablet `RCVD Assistive Tech only on 11/6/2024` - needing device breakdown.
- [ ] **Research Tracker (CAIA Internal):** Final Device and missing information needed `to be added when available`
- [ ] **Review research folder and assets**
- [x] **OFFICIAL RAW RESEARCH NOTES** Create [CAIA notes document](https://docs.google.com/document/d/1lQzJB0q0ePFumxC749K-58lVhaAZh7_OZPhR-uU9Cuk/edit?tab=t.0)
- [ ] **Participate in research** **Date Span:** 11/15/2024 - 11/20/2024
- [ ] **SYNC:** Connect with CAIA A11ys on research findings (meeting)
- [ ] **Create/update the CAIA A11y Research Deliverable** This is dependent on the number of study participants CAIA supports. If minimal, A11Ys may share feedback only in the feedback Mural or this ticket.
- [ ] **Share the completed deliverable** with the project team via Slack: `Primary Team Slack` and `Ticket`
- [ ] **TEMPLATE SESSION NOTES**: Create [CAIA notes document] for a11ys to use during the sessions
```

```[tasklist]
### VFS TEAM Tasks
- [x] Tag @Naomi Eke in Slack [#sitewide-content-accessibility-ia](https://dsva.slack.com/archives/C01K37HRUAH) Link to VA [OCTO Slack Thread 9/18/2024](https://dsva.slack.com/archives/C01K37HRUAH/p1726675576803719)
- [ ] Your team's researchers and observers read through platform guidance on [research with assistive technology users](https://depo-platform-documentation.scrollhelp.site/research-design/research-with-assistive-technology-users) and have also read and learned [how to be respectful of participants with disabilities](https://depo-platform-documentation.scrollhelp.site/research-design/disability-etiquette)
- [ ] Your team's researchers have read through the resources for researchers on the VA Platform's [Accessibility on Va.gov page.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-on-va-gov#AccessibilityonVA.gov-ResourcesforResearchers)
- [ ] [Inclusive Recruitment Strategies](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/research/recruitment.md)
- [x] Consider [creating your prototype in CodePen](https://depo-platform-documentation.scrollhelp.site/research-design/accessible-prototyping-with-codepen) for the most accurate results - `Using Codespace`
- [x] `Recommending Pilot` Review our [assistive technology pilot guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md) and request a pilot, if desired
- [x] Initial meeting with CAIA a11ys `Scheduled`
- [x] Request pilot sessions with CAIA a11ys #95123
- [x] ~Schedule pilot session with CAIA a11ys (Mobile)~
- [x] Schedule pilot session with CAIA a11ys (Desktop)
- [x] Requested Design Review prior to Pilot ([Figma Review 8/28/2024](https://github.com/department-of-veterans-affairs/va.gov-team/issues/90140#issuecomment-2315675462))
- [x] **DATES NEEDED** See `Table` in the ticket above. Please provide date range and slots for AT sessions being planned. `Estimated Start: EOM Oct./Early Nov.` RCVD 11/6/2024 - holds now on calendar
- [x] **ASSISTIVE TECH. INFO NEEDED:** Please provide the AT the users will be utilizing during the research sessions in `Table` in the ticket above. `Provided 11/6/2024`
- [x] **PROTOTYPE:** Provide `CodeSpace` prototype to CAIA A11ys via this ticket in the supporting artifacts section. [Codespace prototype link](https://effective-space-tribble-rq7q67g6gv359p5-3001.app.github.dev/mock-form-ae-design-patterns/2) `Provided 11/6/2024`
- [x] **INVITES:** Forward invites from Perigean to CAIA A11ys (@sara-amanda) `Updated Calendar 11/8/2024`
- [ ] **Device Detail:** Desktop, Mobile and Tablet - Device breakdown needed for session prep
```

```[tasklist]
### Related Tickets
- [ ] #90140
- [x] [Figma Prototype - Initial Review 10/28/2024](https://github.com/department-of-veterans-affairs/va.gov-team/issues/90140#issuecomment-2315675462)
- [ ] #94060
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/95123
```

**Comment by sara-amanda** [comment]
## OCTO Feedback
@artsymartha68 shall we keep the priority level the same as the intake ticket? Assuming yes, but if you can verify. Many thanks! 

#90140

cc: @NaomiPMC

**Comment by sara-amanda** [comment]
# CAIA a11y Research Support

@ChristineRoseSteiffer  _Thank you for submitting a research ticket with CAIA! Our a11ys are looking forward to working with you on your latest research project._ 

## Next Steps
**The following are the initial, next steps in our process:** 

- [ ] Review the refined ticket above. 
- [ ] Review the VFS Team Task list items, that will assist us in supporting you.
- [ ] Let's go ahead and schedule your initial meeting with a CAIA a11y(s) - just send us your availability [in our current Slack thread for 10/16 and 10/18](https://dsva.slack.com/archives/C8E985R32/p1728568230052909?thread_ts=1728507763.664619&cid=C8E985R32) with a tag `@sitewide-caia-accessibility`. 
    - [x] _You can additionally choose to post your availability in this ticket's comments and tag me @sara-amanda._ 
- [ ] If possible, provide your official research dates (start and end) so we can set holds on our calendar. 
    - [x] **Generic Holds Set for 10/28 - 11/8/2024 9 a.m. - 5 p.m. ET.** 
- [x] If available, let us know where you plan to conduct your research: Codespace
- [ ] If available, let us know what assistive technology you are looking to test users with. 
- [ ] If possible, let's go ahead and schedule your [pilot session](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md) with a CAIA a11y, so that we can test your prototype with the AT you will be testing real users with. 

_**We are happy to answer any questions you may have, along the way. Thank you for reaching out to us and we are happy to help!**_ 

cc: @NaomiPMC @artsymartha68 @coforma-jamie

**Comment by ChristineRoseSteiffer** [comment]
Hey folks! I wanted to follow up to see how I can move forward getting some guidance from content folks on the couple of questions we have for them in our study. Is there another ticket I should create for that? Apologies if I missed a step along the way. Thanks for the help! CC @msbtterswrth @kristen101606

**Comment by sara-amanda** [comment]
## Pilot Feedback

Feedback from the pilot session and our notes are [commented and linked in the pilot ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/95123#issuecomment-2465294250). 

#95123 

cc: @ChristineRoseSteiffer and @kristen101606

**Comment by sara-amanda** [comment]
## Calendar Updates

[**CAIA Capacity Calendar**](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/NewYork)

### Invites 
Thank you @ChristineRoseSteiffer and @kristen101606 for sending over the official invites, they have been added to our shared calendar. Our team members, who are covering the sessions, have been added to them as well. 

cc: @coforma-jamie @SarahKay8 @mutiaadhocteam

**Comment by kristen101606** [comment]
Hi @coforma-jamie, Laura, and the entire CAIA team! Now that we've collected feedback from participants in our study on the Update Prefill pattern, we have a clearer idea of what's working and what needs tweaking. We have 7 copy-related questions we wanted to run past you, listed in the the "Post-Study" portion of [our Figma file](https://www.figma.com/design/1z3bAkQl4uR1IvAxmtyqZi/AE-Design-Patterns---Update-Prefill?node-id=689-21949&t=cSyB7WvPzuY0jrJu-1).

We've signed up for [office hours](https://dsva.slack.com/docs/T03FECE8V/F06JTSH7CHZ) on Tues, Dec 3, but are happy to set up another meeting if you think we need more time to chat. You're also welcome to comment directly in our Figma file!

**Comment by sara-amanda** [comment]
**Note**: Closing ticket - and opening ticket to reflect continuing work on the post-study portion. 

[a11y Support] Authenticated Experience Design Patterns Team - Update Prefill Pattern: Post-Study Support (Office Hours Support) #97468
